hydra-node/src/Hydra/Network/Reliability.hs

```diff
@@ -199,11 +199,13 @@ mkMessagePersistence numberOfParties msgPersistence ackPersistence =
     , saveAcks = \acks -> do
         save ackPersistence acks
     , loadMessages = do
-        loadAll msgPersistence
+        loadAll persistenceIncremental
     , appendMessage = \msg -> do
-        append msgPersistence msg
+        append persistenceIncremental msg
     }
 
+-- TODO(Elaine): this can probably be replaced with new persistence
```

Comment:
Must: not change this module.

Similar to the API sever, the network layer just uses `PersistenceIncremental` to store stuff on disk. We should keep it that way and not refactor that in this PR.